integration/build/build_test.go

```diff
@@ -249,6 +254,8 @@ RUN cat somefile`
 // #35403 #36122
 func TestBuildUncleanTarFilenames(t *testing.T) {
 	skip.If(t, versions.LessThan(testEnv.DaemonAPIVersion(), "1.37"), "broken in earlier versions")
+	skip.If(t, testEnv.DaemonInfo.OSType != "linux") // Windows doesn't support FROM scratch
```

Comment:
Comment should probably be passed as third argument.

Also: does this test require `FROM scratch`? Or could we use some other image to make it work on Windows?